torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1416,7 +1501,8 @@ std::vector<CodeGen::CallArg> TensorExprKernel::prepareRunArgs(
   // TODO: preallocate `runArgs` during compilation and fill in values where
   // possible (e.g. for constant tensors)
   std::vector<CodeGen::CallArg> runArgs;
-  runArgs.reserve(inputs.size() + bufOutputs_.size());
+  runArgs.reserve(
+      inputs.size() + input_stride_args_.size() + bufOutputs_.size());
```

Comment:
If the strides here are runtime args, shouldn't that be already part of `inputs`?

Reply:
they're not arguments to the graph, they're taken from the input tensors themselves